tests/debugging/probe/test_registry.py

```diff
@@ -56,6 +56,8 @@ def test_registry_location_error():
     assert not registry.get_pending(__file__)
 
     # Check that we emitted the correct diagnostic error message
+    for e in status_logger.queue:
+        del e["timestamp"]
```

Comment:
why? do that we can still assert equality of a list of dicts?

Reply:
yep, the timestamp is just randomness here, so we get rid of it before the comparison